### PR TITLE
[Util] Parse FLAGS_pd_unittest_use_cinn from python

### PR DESCRIPTION
**Pull request by jiahy0825**
### PR types
Others

### PR changes
Others

### Description

pcard-76996

提供工具函数 `unittest_use_cinn`，判断单测是否进入 cinn 的执行逻辑

### CODE REVIEW

File: test/ir/pir/cinn/utils.py [hunk 1]
```diff
@@ -23,6 +23,12 @@
 __WHILE_OP_NAME = "pd_op.while"
 
 
+def unittest_use_cinn():
+    import os
+
```
Comment (tc20042008) [inline test/ir/pir/cinn/utils.py]:
```python
use_cinn = os.getenv("FLAGS_pd_unittest_use_cinn", False)
true_value_set = {True, 1, "True", "true"}
false_value_set = {False, 0, "False", "false"}
assert use_cinn in (true_value_set | false_value_set)
return use_cinn in true_value_set
```

File: test/ir/pir/cinn/utils.py [hunk 2]
```diff
@@ -23,6 +23,12 @@
 __WHILE_OP_NAME = "pd_op.while"
 
 
+def unittest_use_cinn():
+    import os
```
Comment (cyber-pioneer) [inline test/ir/pir/cinn/utils.py]: 尽量不局部import

**Review by tc20042008 — DISMISSED** [review]

File: test/ir/pir/cinn/utils.py [hunk 3]
```diff
@@ -23,6 +24,14 @@
 __WHILE_OP_NAME = "pd_op.while"
 
 
+def unittest_use_cinn():
+    use_cinn = os.getenv("FLAGS_pd_unittest_use_cinn", False)
+    true_value_set = {True, 1, "True", "true"}
+    false_value_set = {False, 0, "False", "false"}
```
Comment (tc20042008) [inline test/ir/pir/cinn/utils.py]: true_value_set = {True, 1, "1", "True", "true"}

Comment (tc20042008) [inline test/ir/pir/cinn/utils.py]: false_value_set = {False, 0, "0", "False", "false"}

**Pull request closed without merging**